src/Oscar.jl

```diff
@@ -70,94 +70,7 @@ end
 # force trigger recompile when folder changes
 include_dependency("../experimental")
```

Comment:
How about moving the code from line 31 to here, which deals with `experimental`, over to `../experimental/Experimental.jl`?